include/openbabel/mol.h

```diff
@@ -321,6 +326,10 @@ enum HydrogenType { AllHydrogen, PolarHydrogen, NonPolarHydrogen };
     //! \return the size of the smallest ring if a and b are in the same ring, 0 otherwise
     //! \since version 2.4
     int AreInSameRing(OBAtom *a, OBAtom *b);
+    //! \return a pointer to the lattice parameters for periodic systems, or NULL if undefined.
+    //! This is an alternative to
+    //! \verbatim OBUnitCell * pCell = (OBUnitCell * )pmol->GetData(OBGenericDataType::UnitCell);\endverbatim
+    OBUnitCell  *GetPeriodicLattice() const { return _unitCell; }
```

Comment:
No, please no. As you indicate, there's already a way to do this. I don't want to store a _unitCell for every molecule, when only a small % have them. That's the point of the GenericData system

Reply:
Agreed.  A redundant copy of the data can also quickly lead to consistency issues.